src/modules/navigator/navigator_params.c

```diff
@@ -121,11 +121,39 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
  * @value 1 Warn only
  * @value 2 Return mode
  * @value 3 Land mode
+ * @value 4 Position Hold mode
  *
  * @group Mission
  */
 PARAM_DEFINE_INT32(NAV_TRAFF_AVOID, 1);
 
+/**
+ * Set NAV TRAFFIC AVOID RADIUS MANNED
+ *
+ * Defines the Radius where NAV TRAFFIC AVOID is Called
+ * For Manned Aviation
+ *
+ * @unit m
+ * @min 500
+ *
+ * @group Mission
+ */
+PARAM_DEFINE_FLOAT(NAV_TRAFF_A_RADM, 500);
+
+/**
+ * Set NAV TRAFFIC AVOID RADIUS
```

Comment:
@LowOrbitIonCannon Can we have a new PR to change this to more natural formatting?

... following pattern as I put above